pkg/agent/monitortool/monitor.go

```diff
@@ -43,10 +43,11 @@ import (
 var icmpEchoID = rand.Int31n(1 << 16)
 
 const (
-	ipv4ProtocolICMPRaw = "ip4:icmp"
-	ipv6ProtocolICMPRaw = "ip6:ipv6-icmp"
-	protocolICMP        = 1
-	protocolICMPv6      = 58
+	ipv4ProtocolICMPRaw      = "ip4:icmp"
+	ipv6ProtocolICMPRaw      = "ip6:ipv6-icmp"
+	protocolICMP             = 1
+	protocolICMPv6           = 58
+	MinReportIntervalSeconds = 10
```

Comment:
there is no need to export this constant

also it feels like this could be typed as a duration: `const minReportInterval = 10*time.Second`